content/blog/multi-lang-components/index.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "Pulumi expands EKS support to Python, Go, and .NET"
+
+# The date represents the post's publish date, and by default corresponds with
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2020-11-19T16:48:16-07:00
+
+# Draft posts are visible in development, but excluded from production builds.
+# Set this property to `false` before submitting your post for review.
+draft: true
+
+meta_desc: Pulumi now supports EKS across Python, Go, .NET, and TypeScript.
+meta_image: meta.png
+authors:
+    - levi-blackstone
+tags:
+    - eks
+    - .net
+    - python
+    - go
+    - typescript
+---
+
+Today we're excited to announce support for one of our most highly-requested features: multi-language components! Our
+first multi-language component is the popular EKS package, which was previously only available for TypeScript. With the
+[v0.21.0 release] (TODO link), teams can manage EKS clusters in their language of choice (Python, .NET, Go, and TypeScript)!
```

Comment:
I don't think dotnet and Go will be in the v0.21 release will it? There's only Python setup in place at this point in our builds